tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1398,12 +1398,12 @@ class CompletionSuite extends BaseCompletionSuite {
            |implicitConversions scala.languageFeature
            |postfixOps scala.languageFeature
            |""".stripMargin,
-      ">=3.0.0" ->
+      ">=3.3.0" ->
         """|dynamics scala.languageFeature
            |existentials scala.languageFeature
            |experimental scala.languageFeature
-           |higherKinds scala.languageFeature
            |implicitConversions scala.languageFeature
+           |postfixOps scala.languageFeature
```

Comment:
Not 100% sure why the change was needed, but I would assume some test classpath issues, which are not really that important